src/read.c

```diff
@@ -367,6 +371,66 @@ static uint32_t avifCodecConfigurationBoxGetDepth(const avifCodecConfigurationBo
     return 8;
 }
 
+#if defined(AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM)
+uint8_t avifCodecConfigurationBoxGetSubsamplingType(const avifCodecConfigurationBox * av1C, uint8_t channelIndex)
+{
+    // See ISO/IEC 23008-12:2024/CDAM 2:2025 section 6.5.6.3.
```

Comment:
Is this the right reference? this method deals with the av1C box, not sure this bit of the spec is relevant, but maybe it's because I don't have the latest version of the spec